index.html

```diff
@@ -289,6 +290,92 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div id="contact-me">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mrgrnekq" method="POST">
+          <input
+            name="user_name"
+            type="text"
+            placeholder="Full name"
+            maxlength="30"
+          />
+          <input name="user_email" type="text" placeholder="Email address" />
```

Comment:
- [ ] Kindly make sure that the email field accepts only emails

Reply:
- [ ] Also, Kindly make sure that a user cannot submit the form if the field is empty

Reply:
Ok, I'll do that. 👍